src/index/service.js

```diff
@@ -61,3 +63,10 @@ const excerptToHtml
 const formatError
   = err =>
     '<p>Unable to fetch blog posts at this time.</p><!-- ' + err + '-->'
+
+const transformTeam 
+  = (teamMembersObj) => {
+    const id = "owl-team"
```

Comment:
IIUC, this looks you're searching for a solution to the mismatch between component-driven design and the original template's page-driven design. (The scripts.js file is still page-driven.)  Except for edge cases, ids should stay out of components.  What do you think about replacing the `$(#owl-team)` jquery search with one that uses classes?